api/src/routes/adminRoutes.ts

```diff
@@ -8,4 +8,6 @@ export const router = express.Router();
 
 router.get('/admin/allUsers', middleware.authorize(Scope.ReadAllUsersRoles), controller.getAllUsersInfo);
 
+router.delete('/admin/userRole', middleware.authorize(Scope.DeleteUsersRole), controller.deleteRole);
```

Comment:
Shouldn't this be in `/users/{user}/roles/{role}`?

Reply:
Hmm maybe? I'm not really sure the proper structure for this. My rationale for putting this under admin is that only admin can delete a role. But since it's properly scoped it shouldn't matter and would probably be best under `/users`. I'm not totally sure tbh, if you think I should put it under that endpoint then I can change it

Reply:
It is more RESTFUL if we put it in `/users/{user}/roles/{role}` and it does follow `endpoints.md` but given the time constraint I'm also OK putting that here

Reply:
Decided to just put it under the endpoint specified, rather do it right now than fix it later haha